hub/constraints/instruction-handling/call/precompiles/flag_sums_and_NSRs.lisp

```diff
@@ -184,42 +184,40 @@
-                                                            (shift    PEEK_AT_CONTEXT          precompile-processing---MODEXP-context-row---FKTR---row-offset          )
-                                                            )
-  )
-(defun    (precompile-processing---nsr-MODEXP-success)    (+
-                                                            (shift    PEEK_AT_SCENARIO         0                                                                             )
-                                                            (shift    PEEK_AT_MISCELLANEOUS    precompile-processing---MODEXP-misc-row---cds---row-offset                    )
-                                                            (shift    PEEK_AT_MISCELLANEOUS    precompile-processing---MODEXP-misc-row---extract-bbs---offset                )
-                                                            (shift    PEEK_AT_MISCELLANEOUS    precompile-processing---MODEXP-misc-row---extract-ebs---row-offset            )
-                                                            (shift    PEEK_AT_MISCELLANEOUS    precompile-processing---MODEXP-misc-row---extract-mbs---row-offset            )
-                                                            (shift    PEEK_AT_MISCELLANEOUS    precompile-processing---MODEXP-misc-row---extract-raw-lead---row-offset       )
-                                                            (shift    PEEK_AT_MISCELLANEOUS    precompile-processing---MODEXP-misc-row---pricing---row-offset                )
-                                                            (shift    PEEK_AT_MISCELLANEOUS    precompile-processing---MODEXP-misc-row---copy-inputs-base---row-offset       )
-                                                            (shift    PEEK_AT_MISCELLANEOUS    precompile-processing---MODEXP-misc-row---copy-inputs-exponent---row-offset   )
-                                                            (shift    PEEK_AT_MISCELLANEOUS    precompile-processing---MODEXP-misc-row---copy-inputs-modulus---row-offset    )
-                                                            (shift    PEEK_AT_MISCELLANEOUS    precompile-processing---MODEXP-misc-row---full-result-transfer---row-offset   )
-                                                            (shift    PEEK_AT_MISCELLANEOUS    precompile-processing---MODEXP-misc-row---partial-result-copy---row-offset    )
-                                                            (shift    PEEK_AT_CONTEXT          precompile-processing---MODEXP-context-row---success---row-offset             )
-                                                            ))
-;; MODEXP flag sum shorthands
-(defun    (precompile-processing---flag-sum-MODEXP-FKTR)       (precompile-processing---flag-sum-standard-failure))
-(defun    (precompile-processing---flag-sum-MODEXP-success)    (precompile-processing---flag-sum-standard-success))
```

Comment:
In the above there was a mix-up: shorthands that were pointing to `flag-sum`'s were labeled as `nsr` and the flag sums were completely wrong (wrt `MODEXP`)